src/main/python/rerank/scripts/build_microblog_dataset_castor_style.py

```diff
@@ -0,0 +1,45 @@
+import json
+from tqdm import tqdm
+import string
+import argparse
+
+from utils import *
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--method', default='QL', help='[QL, QL+RM3]')
+args = parser.parse_args()
+
+
+searcher11 = JSearcher(JString('/tuna1/indexes/lucene-index.mb11.pos+docvectors+rawdocs'))
```

Comment:
Add ability to pass in index locations from cmdline args, but with `/tuna` as default is okay.

Reply:
Thanks.